File: C_Programming/Study/Q4_Assignment9.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  CountMult
// Output:          
// Description:     Assignment_9: Q.4: Write a program which accept Number & return the multiplication of all digits. 
//                                     
// Date:            30/05/2026
// Author:          Shital Ajit Nikam   
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////


int CountMult(int iNo)
{
    int iDigit = 0;
    int iMul = 1;

   
    while(iNo != 0)
    {
        iDigit = iNo % 10;
        iMul = iMul * iDigit;

        iNo = iNo /10;
    }
    
    return  iMul;
}

int main()
{
    int iValue = 0;
    int iRet = 0;
        
    printf("Enter Number: \n");
    scanf("%d", &iValue);

    iRet = CountMult(iValue);

    printf("Multiplication of Digits is : %d",iRet);

    return 0;
}

/////////////////////////////////////////////////////////////////////////////////////
//Time Complexity : O(N)  where N = length of Number
/////////////////////////////////////////////////////////////////////////////////////
// Input : 922432
// Output : 864  
// Input : 2395
// Output : 270
// 
/////////////////////////////////////////////////////////////////////////////////////

File: Python_Programming/Competative/Q4_Assignment9.py
#//////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name :  Cube
#        
# Description:     Assignment_9: Q.4: Write a program which accept one number & prints cube of that number.
# Input  :         5
# Output :         125
#                                     
# Date:            30/06/2026
# Author:          Shital Ajit Nikam   
#
#//////////////////////////////////////////////////////////////////////////////////////////////////////////

def Cube(No1):
    Square = 0
    Square = No1 ** 3               #Cube of number (No1 ^ 3)
    return Square
    
def main():
    value1 = int(input("Enter the number: "))
        
    Ret = Cube(value1)   
    
    print(f"Square of {value1} is {Ret}")   
     
    
if __name__ == "__main__":
    main()